upup/pkg/fi/cloudup/vspheretasks/attachiso.go

```diff
@@ -57,6 +94,73 @@ func (_ *AttachISO) CheckChanges(a, e, changes *AttachISO) error {
 }
 
 func (_ *AttachISO) RenderVC(t *vsphere.VSphereAPITarget, a, e, changes *AttachISO) error {
-	glog.Info("AttachISO.RenderVC invoked!")
+	startupScript, err := changes.BootstrapScript.ResourceNodeUp(changes.IG)
+	startupStr, err := startupScript.AsString()
+	if err != nil {
+		return fmt.Errorf("error rendering startup script: %v", err)
+	}
+	dir, err := ioutil.TempDir("", *changes.VM)
+	defer os.RemoveAll(dir)
+
+	isoFile := createISO(changes, startupStr, dir)
```

Comment:
If there is an error in createISO we fail the execution by invoking glog.Fatalf method. Basically there is no point in continuing if they fail.